packages/vuetify/src/components/VTable/_variables.scss

```diff
@@ -0,0 +1,33 @@
+@use 'sass:math';
+@use 'sass:map';
+@use '../../styles/settings';
+@use '../../styles/tools';
+
+$table-compact-header-height: 32px !default;
+$table-compact-row-height: $table-compact-header-height !default;
+$table-comfortable-header-height: 40px !default;
+$table-comfortable-row-height: $table-comfortable-header-height !default;
+$table-default-header-height: 48px !default;
+$table-default-row-height: $table-default-header-height !default;
```

Comment:
Density scales everywhere else are something like `(0, -2, -4)`, which is then multiplied by `$spacer` for each level and added to the base height.